### DISCUSSION TITLE
Add support for showing unread message counts

### DISCUSSION THREAD

**Discussion by Kabouik** [d99kris/nchat#154] [Ideas]
I believe I suggested that once on the Telegram group, but as per our discussion today, I'm opening an issue here to cĺarify and help keeping track of feature requests. As an option, it would be useful to show the number of unread messages/mentions/reactions (ideally with different symbols for each) in the list and possibly the main view too (where it might be easier to fit more details). This would go well with `Jump to mention/reaction` keybinds.

**Comment by d99kris** [comment]
Hi @Kabouik,

Thanks for reminding me about this feature request and reporting it here. I think it's a good idea. I'm currently a bit busy, but I will try to look into it soon.

In any case, I encourage anyone else interested in this feature to upvote it. This will help gauge the level of interest.

**Comment by d99kris** [comment]
Once/if this gets addressed, the feature request #183 is similar in that it would use same backend data, which is currently not available.

**Comment by d99kris** [comment]
This is not trivial to support given nchat's current design lazy fetching messages from latest/newest available. Will mark `not-planned` for now.
